src/NzbDrone.Core/Datastore/Migration/038_remove_plex_hometheatre.cs

```diff
@@ -0,0 +1,15 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(38)]
```

Comment:
Does fluentmigrator care if this is 038 or just 38? Noticed we have some both ways. Wouldn’t think it would matter given it’s an int value, but just a note from consistency standpoint.

Reply:
I've stripped the leading zeros from the few that had them